src/utils/index.js

```diff
@@ -2,5 +2,4 @@
  * Sanitizes a given text
  * @param {String} text
  */
-export const sanitizeText = (text) => text.trim();
-export const sanitizeText2 = (text) => text.trim();
+export const sanitizeText = (text) => (text ? text.trim() : '');
```

Comment:
I think wrapper parenthesis are not necessary.

Reply:
lint rule: 
<img width="817" alt="Screen Shot 2020-02-20 at 12 50 27 PM" src="https://user-images.githubusercontent.com/59701575/74963363-97558c80-53df-11ea-87eb-322d6c7f2168.png">